data/aerial-photography/index.html

```diff
@@ -75,9 +76,9 @@ <h3 id="NAIP">
     <ul class="dotless">
       <li>Natural Color (RGB)
       <li>False Color (CIR)
-      <li>~1 meter pixels
+      <li>~1-meter pixels
     </ul>
-    <p><a href="https://www.fsa.usda.gov/programs-and-services/aerial-photography/imagery-programs/naip-imagery/index">National Agricultural Imagery Program</a> is color aerial photography typically collected ever 2 or 3 years in the middle of the summer. Starting in 2006 the acquisitions have been statewide. The resolution is usually 1 meter with a horizontal positional accuracy of 5 meters. In addition to the 3-band Red Green Blue (RGB) color imagery, there is a color-infrared dataset from 2006, and a 3-band and 4-band dataset from 2009. The 2011 dataset is 4-band for the TIFF images and 3-band for the county moasics. The 2014 dataset is 3-band with separate single band image tiles for the 4th band for the TIFF images, the county moasics are 3-band. Currently there is NAIP imagery from 2003, 2004, 2006, 2009, 2011, 2014, and 2016.</p>
+    <p><a href="https://www.fsa.usda.gov/programs-and-services/aerial-photography/imagery-programs/naip-imagery/index">National Agriculture Imagery Program</a> is color aerial photography typically collected every 2 or 3 years in the middle of the summer. Starting in 2006 the acquisitions have been statewide. The resolution is usually 1 meter with a horizontal positional accuracy of 5 meters. In addition to the 3-band RGB color imagery, there is a color-infrared dataset from 2006 and a 3-band and 4-band dataset from 2009. The 2011 dataset is 4-band for the TIFF images and 3-band for the county moasics. The 2014 dataset is 3-band with separate single-band image tiles for the 4th band for the TIFF images, and the county moasics are 3-band. Currently there is NAIP imagery from 2003, 2004, 2006, 2009, 2011, 2014, and 2016.</p>
```

Comment:
(First sentence)--"color aerial photography" of what? Of "select areas in Utah" again?

Reply:
all references to "National Agricultural Imagery Program" should be changed to "National Agriculture Imagery Program"

Should be ~1-foot or ~6-inch pixels

Reply:
Sorry, what should be ~1-foot or ~6-inch pixels? And what about the question for that first sentence?

Reply:
NAIP covers all of Utah. I'm not sure what is up with the ~1-foot or ~6-inch thing. @rkelson what are you talking about?